### DISCUSSION TITLE
Lots of slatejs editors on one page

### DISCUSSION THREAD

**Discussion by nc** [ianstormtaylor/slate#4371] [Q&A]
Hey, i'm building an app which needs rich text notes (think sticky notes) and am wondering if Slate would be a good fit for making the text in each note customizable (you know bold, italics, maybe a couple of different font-sizes). Is having 50-100 slate js editors rendered on a page going to be problematic, or no issue - thought I'd ask here first. Thanks :)

**Comment by risenxxx** [comment]
You may serialize the content into the HTML and render it as is until user starts editing. Then, after the editing was completed, serialize it again.
